tools/interactive/interactivetool_anylabeling.xml

```diff
@@ -0,0 +1,83 @@
+<tool id="interactive_tool_anylabeling" tool_type="interactive" name="AnyLabeling Interactive" version="@VERSION@" profile="24.2">
+    <description>Effortless data labeling with AI support from YOLO and Segment Anything!</description>
+    <icon src="anylabeling.png"/>
+    <requirements>
+        <container type="docker">quay.io/bgruening/anylabeling:0.4.30</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="AnyLabeling" requires_domain="True">
+            <port>5800</port>
+        </entry_point>
+    </entry_points>
+    <command detect_errors="exit_code">
+    <![CDATA[
+         export HOME=\$PWD &&
+         
+         mkdir -p ./home ./home/output ./home/input_images &&
+	 chown -R 1000:1000 ./home &&
```

Comment:
why 1000?

Reply:
It is from the docker base image. It's the default ID of the user the application runs as. I saw the ilastik and other IT also have this

Reply:
Just wanted to make sure it's needed.